storage/engine/rocksdb/db.cc

```diff
@@ -1089,6 +1130,8 @@ DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
     table_options.block_cache = rocksdb::NewLRUCache(
         block_cache_size, num_cache_shard_bits);
   }
+  table_options.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10));
+  table_options.format_version = 2;
```

Comment:
Does changing the format do anything besides (presumably) enabling the prefix-based bloom filter? Is it worth benchmarking this change separately?